practice1/HomeWork3/Users/Person.cs

```diff
@@ -22,19 +21,13 @@ public Person()
             JobTitle = new Bogus.DataSets.Name().JobType();
             JobDescription = new Bogus.DataSets.Name().JobTitle();
             JobSalary = new Bogus.Randomizer().Int(500, 10000);
-        }
-
-        public string FullName
-        {
-            get
-            {
-                return $"{FirstName} {LastName}";
-            }
+            FullName = $"{FirstName} {LastName}";

```

Comment:
It would be better, if the FullName will be as lambda. In case of changing the FirstName or LastName, the FullName will remain the same